### PR TITLE
feat(ui-ux): EVM (TransferDomain and changi network) (#1797)

### PR DESCRIPTION
**Pull request by lykalabrada**
* feature(ui-ux): added parser for transferbalance txn type

* updated transfer balance to transferdomain

* updated UI

* added getDecodedAddress function

* added evm compatable docker image

* added getDecodedAddress

* added TransferDomainType as enum

* fixed alignment in docker file

* DfTxTransferDomain txn fix

* updated Ui

* hide link incase of eth address in local and devnet and show balance for eth address

* update walletkit version

* type fix

#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link:

<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,10 +24,10 @@
...
     "@reduxjs/toolkit": "^1.9.5",
     "@tailwindcss/line-clamp": "^0.4.4",
-    "@waveshq/standard-defichain-jellyfishsdk": "^1.9.0",
-    "@waveshq/standard-web": "^1.9.0",
-    "@waveshq/walletkit-core": "^1.1.0",
-    "@waveshq/walletkit-ui": "^1.1.0",
+    "@waveshq/standard-defichain-jellyfishsdk": "^2.2.0",
+    "@waveshq/standard-web": "^2.2.0",
+    "@waveshq/walletkit-core": "^1.3.1",
+    "@waveshq/walletkit-ui": "^1.3.1",
```
Comment (pierregee) [inline package.json:30]: Need to update to 1.3.2

File: src/pages/transactions/_components/DfTx/DfTxTransferDomain.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  decodedAddress: EthDecodedAddress | undefined;
+  testId: string;
+}): JSX.Element {
+  const { connection } = useNetwork();
+  // hide link in case of ETH address and network is not MainNet/TestNet
+  if (
+    decodedAddress?.type === AddressType.ETH &&
+    ![EnvironmentNetwork.MainNet, EnvironmentNetwork.TestNet].includes(
+      connection
+    )
```
Comment (pierregee) [inline src/pages/transactions/_components/DfTx/DfTxTransferDomain.tsx:92]: Do we still need this?

File: src/store/metachain.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+export function getBaseUrl(network: EnvironmentNetwork): string {
+  switch (network) {
+    case EnvironmentNetwork.MainNet:
+    case EnvironmentNetwork.TestNet:
+      return "https://changi.ocean.jellyfishsdk.com";
+    default:
+      throw new Error("No api endpoint available");
+  }
+}
```
Comment (pierregee) [inline src/store/metachain.tsx:32]: Need to update now that changi is available

File: src/pages/transactions/_components/DfTx/DfTxTransferDomain.tsx (hunk 2 above)
Comment (fullstackninja864) [reply src/pages/transactions/_components/DfTx/DfTxTransferDomain.tsx:92]: Until we don't have evm indexer for mainnet and testnet lets leave this ?

**Review by fullstackninja864 — APPROVED** [review]

**Pull request merged**